tests/h/search/query_test.py

```diff
@@ -97,6 +97,27 @@ def search(self, search):
 
 
 class TestGroupAuthFilter(object):
+    def test_does_not_return_annotations_if_group_not_readable_by_user(self,
+            search, Annotation, group_service):
+        group_service.groupids_readable_by.return_value = []
+        Annotation(groupid="group2").id
+        Annotation(groupid="group1").id
+        Annotation(groupid="group1").id
+
+        result = search.run({})
+
+        assert not result.annotation_ids
+
+    def test_returns_annotations_if_group_readable_by_user(self,
+            search, Annotation, group_service):
+        group_service.groupids_readable_by.return_value = ["group1"]
+        Annotation(groupid="group2", shared=True).id
+        users_private_ids = [Annotation(groupid="group1").id,
```

Comment:
These could be the users private IDs, but they could also be annotations in a private group that the user is a member of. Perhaps just rename this `expected_ids`?